src/tests/topp/TextExporter_2_input.consensusXML

```diff
@@ -10,1164 +10,1325 @@
 		</map>
 	</mapList>
 	<consensusElementList>
-		<consensusElement id="e_0" quality="1">
-			<centroid rt="2134.18" mz="1097.65" it="1.29711e+09"/>
+		<consensusElement id="e_0" quality="1.0">
+			<centroid rt="2134.179999999999836" mz="1097.650000000000091" it="1.29711e09"/>
 			<groupedElementList>
-				<element map="0" id="0" rt="2135.53" mz="1097.29" it="7.95801e+08"/>
-				<element map="1" id="0" rt="2132.83" mz="1098.01" it="1.79841e+09"/>
+				<element map="0" id="0" rt="2135.5300000000002" mz="1097.289999999999964" it="7.95801e08"/>
+				<element map="1" id="0" rt="2132.829999999999927" mz="1098.009999999999991" it="1.79841e09"/>
 			</groupedElementList>
+			<UserParam type="string" name="best ion" value="[M+H]+"/>
 		</consensusElement>
-		<consensusElement id="e_1" quality="1">
-			<centroid rt="1668.34333333333" mz="501.195333333333" it="4.25668e+08"/>
+		<consensusElement id="e_1" quality="1.0">
+			<centroid rt="1668.343333333330065" mz="501.195333333332997" it="4.25668e08"/>
 			<groupedElementList>
-				<element map="0" id="0" rt="1650.21" mz="501.24" it="3.5816e+08"/>
-				<element map="1" id="0" rt="1622.51" mz="501.126" it="6.96541e+08"/>
-				<element map="2" id="0" rt="1732.31" mz="501.22" it="2.22304e+08"/>
+				<element map="0" id="0" rt="1650.210000000000036" mz="501.240000000000009" it="3.5816e08"/>
```

Comment:
Do you know why these now have that many digits?

Reply:
Changed that in the recent commit.